app/views/layouts/application.html.erb

```diff
@@ -58,9 +58,26 @@
 <% end %>
 
 <% content_for :footer_support_links do %>
-  <li>
-    <%= link_to(t('.staff_guide'), staff_path) %>
-  </li>
+  <ul>
+    <li>
+      <%= link_to(t('.staff_guide'), '/staff/staffuiguide.md') %>
+    </li>
+    <li>
+      <%= link_to(t('.recent_changes'), '/staff/changes.md') %>
+    </li>
+    <li>
+      <%= link_to(t('.stats'), 'https://staff.prisonvisits.service.gov.uk/en/metrics') %>
+    </li>
+    <li>
+      <%= link_to(t('.contact_us'), 'https://www.prisonvisits.service.gov.uk/en/feedback/new?referrer=staff') %>
```

Comment:
Same here, how were we linking to the public feedback page before? This link will change soon anyway when we have the contact page.